heron/common/src/cpp/basics/mempool.cpp

```diff
@@ -21,5 +21,5 @@
 
 // TODO(nlu): get the pool size limit from config
 MemPool<google::protobuf::Message>* __global_protobuf_pool__ =
-                                   new MemPool<google::protobuf::Message>(50 * 1024 * 1024);
+                                   new MemPool<google::protobuf::Message>(512);
```

Comment:
This is explained above: https://github.com/twitter/heron/pull/2073#issue-243279094